onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4208,19 +4208,54 @@ def aten_native_dropout_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::native_group_norm", trace_only=True)
```

Comment:
Add a comment on why trace_only is needed?

Reply:
Adding checking some None input so need trace_only=True